### DISCUSSION TITLE
Better https://avaloniaui.github.io/icons.html page

### DISCUSSION THREAD

**Discussion by CzBuCHi** [AvaloniaUI/Avalonia#14929] [Ideas]
I was annoyed with state of https://avaloniaui.github.io/icons.html page - specially that some SVG images are missing and its pita to look for good icon (when you dont know its name) 

So i made my own version: [Avalonia Icons.html.txt](https://github.com/AvaloniaUI/Avalonia/files/14572426/Avalonia.Icons.html.txt) (github is not allowing me to upload html file directly, so it has txt extension)

Page is showing only embeded SVG images, that when clicked will copy corresponding &lt;StreamGeometry&gt; to clipboard ..

Im placing it here, in case someone else is also annoyed....

Feel free to use it, burn it, or anything else ...

**Comment by timunie** [comment]
@CzBuCHi probably you want to file a PR to https://github.com/jmacato/fluentui-system-icons which would be very appreciated

/cc @jmacato 

> [!NOTE]
> I've updated the repo branch

**Comment by CzBuCHi** [comment]
will doo ...

edit: or not .... i have no idea how icnos.html is generated ..
